### ISSUE TITLE
Command line argument parsing is buggy

### ISSUE THREAD

**Issue by smaslennikov** [wata727/tflint#99]
Seems that if I give `tflint` and invalid option such as `aws_region`, placing it on the tail, the wrong argument is listed as wrong:

```bash
$ tflint --debug --deep --aws_region us-east-1
ERROR: `--debug` is unknown options. Please run `tflint --help`
```

I haven't looked at the code handling this, maybe you meant to list all the command line arguments in that error message?

Lastly, I'll quietly mention that "X is unknown options" isn't quite proper grammar usage. 

Utterly minor bug I hate even reporting.

**Comment by wata727** [comment]
Oh, sorry. I fixed this bug.
In addition, since I'm not good at English, I think that there are many unnatural parts.
If you notice something, please send me a patch :)

**Comment by smaslennikov** [comment]
That's awesome, thank you!
